ccm 16.6.0-rc4 has missing dependencies on SL5

Trying to deploy 16.6.0-rc4 on a SL5 system, I got the following `ncm-spma` error:
```
2016/07/17-20:23:49 [ERROR] Failed synchronisation with upstream: Error: Package: ccm-16.6.0-rc4_1.noarch (quattor_16.6.0-rc4)
           Requires: perl(JSON::XS) >= 2.3.0
           Installed: perl-JSON-XS-2.30-1.el5.rf.x86_64 (installed)
               perl(JSON::XS) = 2.3
           Available: perl-JSON-XS-1.43-1.el5.x86_64 (sl5_epel)
               perl(JSON::XS) = 1.43

```
I am not completely sure why the installed version is considered lower than 2.3.0... It would be good to put the non standard dependency in `quattor_externals` YUM repo.

This is a showstopper for [16.6.0-rc4](https://github.com/quattor/release/issues/253) release.

--- a/pom.xml
+++ b/pom.xml
@@ -278,7 +278,9 @@
         <requires>
           <require>perl(Crypt::SSLeay)</require>
           <require>perl(LWP::Protocol::https)</require>
-        </requires>
+          <!-- perl-JSON-XS rpm dependency set explicitly; EL5 metadata doesn't play nicely with 'perl(JSON::XS) >= 2.3.0'
+               set by 'use perl-JSON-XS 2.3.0' and version requirement is missing if 'use perl-JSON-XS v2.3.0' is used -->
+          <require>perl-JSON-XS &gt;= 2.3.0</require>        </requires>
         <mappings>
           <mapping>
             <directory>/usr/lib/perl/EDG/WP4/CCM</directory>

--- a/src/main/perl/JSONProfileSimple.pm
+++ b/src/main/perl/JSONProfileSimple.pm
@@ -31,7 +31,7 @@ use strict;
 use warnings;
 
 use EDG::WP4::CCM::Fetch qw(ComputeChecksum);
-use JSON::XS 2.3.0;
+use JSON::XS v2.3.0;
 
 $SIG{__DIE__} = \&confess;
 

--- a/src/main/perl/JSONProfileTyped.pm
+++ b/src/main/perl/JSONProfileTyped.pm
@@ -52,7 +52,7 @@ use strict;
 use warnings;
 
 use EDG::WP4::CCM::Fetch qw(ComputeChecksum);
-use JSON::XS 2.3.0;
+use JSON::XS v2.3.0;
 use B;
 use Scalar::Util qw(blessed);
 

